src/components/Tokens/TokenDetails/TokenDetail.tsx

```diff
@@ -14,14 +14,14 @@ import { useAtomValue } from 'jotai/utils'
 import { darken } from 'polished'
 import { Suspense, useCallback } from 'react'
 import { useState } from 'react'
-import { ArrowLeft, Heart } from 'react-feather'
+import { ArrowLeft } from 'react-feather'
 import { useNavigate } from 'react-router-dom'
 import styled from 'styled-components/macro'
-import { ClickableStyle, CopyContractAddress } from 'theme'
+import { CopyContractAddress } from 'theme'
 import { formatDollarAmount } from 'utils/formatDollarAmt'
 
-import { favoritesAtom, filterNetworkAtom, useToggleFavorite } from '../state'
-import { ClickFavorited } from '../TokenTable/TokenRow'
+import { filterNetworkAtom, useIsFavorited, useToggleFavorite } from '../state'
+import { ClickFavorited, FavoriteIcon } from '../TokenTable/TokenRow'
```

Comment:
Consider exporting a single \<TokenFavoriter /> component from TokenRow to get rid of duplicate code

Reply:
oh whoops didn't realize this was already merged

Reply:
lmao no worries, if I touch this file again I'll update